### PR TITLE
3775 fix geocode shared table rw

### LINKED ISSUE #3775: Can't geocode a table shared with me
If I try to geocode a table shared with me (write permissions) [it fails](https://rollbar.com/vizzuality/CartoDB/items/7041/). Not sure if this is actually the desired behaviour.

Happens in both old and new endpoints.

cc @Kartones @rafatower @iriberri

### PR DESCRIPTION
**Pull request by rafatower**
This fixes #3775 Can't geocode table shared with me (with write permissions). At first I tried to fix it by granting sufficient permissions to the user, but turns out it is a [limitiation of postgres](https://github.com/CartoDB/cartodb/issues/3775#issuecomment-122012965).

Sbd from @CartoDB/backend please review

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5071/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5076/))

File: app/models/geocoding.rb [hunk 1]
```diff
@@ -122,6 +122,9 @@ def run!
     rows_geocoded_before = table_service.owner.in_database.select.from(table_service.sequel_qualified_table_name).where(cartodb_georef_status: true).count rescue 0
 
     self.run_geocoding!(processable_rows, rows_geocoded_before)
+  rescue => e
+    CartoDB.notify_exception(e)
```
Comment (juanignaciosl) [inline app/models/geocoding.rb]: Could you add context information (table, user...)?

**Comment by juanignaciosl** [comment]
:+1:

File: app/models/geocoding.rb (hunk 1 above)
Comment (rafatower) [inline app/models/geocoding.rb]: I'll move all that code, when I put a proper logger #4417

**Pull request merged**